packages/shorebird_cli/test/src/commands/patch/patch_ios_command_test.dart

```diff
@@ -1321,6 +1324,66 @@ Please re-run the release command for this version or create a new release.'''),
               ).called(1);
             },
           );
+
+          test(
+            'prints the file location when linking is less than the minimum',
+            () async {
+              when(
+                () => aotTools.link(
+                  base: any(named: 'base'),
+                  patch: any(named: 'patch'),
+                  analyzeSnapshot: any(named: 'analyzeSnapshot'),
+                  genSnapshot: any(named: 'genSnapshot'),
+                  kernel: any(named: 'kernel'),
+                  workingDirectory: any(named: 'workingDirectory'),
+                  outputPath: any(named: 'outputPath'),
+                  dumpDebugInfoPath: any(named: 'dumpDebugInfoPath'),
+                ),
+              ).thenAnswer(
+                (_) async => 20,
```

Comment:
I presume 20 is below the threshold required to trigger this warning? It'd be good to have named constants throughout to make this clearer.